tests/wasi.rs

```diff
@@ -130,10 +130,13 @@ impl crate::wasi_snapshot_preview1::WasiSnapshotPreview1 for WasiCtx {
         let mut bc = GuestBorrows::new();
         let mut slices: Vec<&'_ mut [u8]> = Vec::new();
         for iov_ptr in iovs.iter() {
```

Comment:
I think just before this you'll want `bc.borrow_slice(iovs)`, but otherwise looks good!

Reply:
Done, thanks